src/driver/kvm.rs

```diff
@@ -229,6 +235,11 @@ impl<T: KVMIntrospectable> Introspectable for Kvm<T> {
                         new,
                         old,
                     },
+                    KVMiEventType::Msr { msr_type, new, old } => EventType::Msr {
+                        msr_type,
+                        new,
+                        old,
+                    },
                     KVMiEventType::PauseVCPU => panic!("Unexpected PauseVCPU event. It should have been popped by resume VM. (Did you forget to resume your VM ?)"),
                     _ => unimplemented!()
```

Comment:
I think I did that temporarily so it compiles. Now that you have added all possible match arms you can just remove this again.